nodes/src/nodes/tool_bland_ai/IGlobal.py

```
@@ -0,0 +1,102 @@
+            )
+        except Exception as e:
+            warning(str(e))
+            raise
+
+    def validateConfig(self) -> None:
+        try:
+            cfg = Config.getNodeConfig(self.glb.logicalType, self.glb.connConfig)
+            api_key = str(cfg.get('apikey') or '').strip()
+            if not api_key:
+                warning('API key is required')
+        except Exception as e:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)